devito/ops/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""
+The ``core`` Devito backend is simply a "shadow" of the ``base`` backend,
```

Comment:
The ``ops`` Devito backend -- and then provide a suitable description, remarking that this is work in progress